content/sigs/platform/index.adoc

```diff
@@ -65,28 +65,70 @@ on topics related to Cloud-Native platforms like Kubernetes or Docker.
 ARM architecture support, adapting RedHat packaging to best practices, etc.)
 * Reviewing JEPs submitted in the area
 
-=== Ongoing projects
+== Projects
 
-* Multi-architecture Docker images
-** Enabling official images to run on Arm, IBM s390x, and other platforms
-* Rework of Windows installers
-** New version of installers for 64bit platforms
-** Removal of Java bundling
-** Creating an official link:https://chocolatey.org/packages/jenkins[Jenkins Chocolatey package]
+This section lists the key initiatives being handled by the Platform SIG.
+See the link:https://docs.google.com/document/d/1bDfUdtjpwoX0HO2PRnfqns_TROBOK8tmP6SgVhubr2Y/edit?usp=sharing[SIG meeting notes] for more information about the ongoing projects.
+See the link:/project/roadmap[Jenkins Project roadmap] for a 
+
+=== Docker images
+
+Jenkins project ships official master and agent images,
+and we would like to offer Wide support of platforms and architectures there.
+Scope of interest:
+
+* Enabling official images to run on Arm, IBM s390x, and other platforms
+* Official master and agent images for Windows
+* Support Multi-architecture Docker images
 * Enabling continuous delivery for Jenkins packaging
 ** Experimental DockerHub organization and deployments from ci.jenkins.io (jep:217[])
 
-See the link:https://docs.google.com/document/d/1bDfUdtjpwoX0HO2PRnfqns_TROBOK8tmP6SgVhubr2Y/edit?usp=sharing[SIG meeting notes] for more information about the ongoing projects.
+=== Plugin management
```

Comment:
Is it intentional that plugin management is listed as a platform project but is not in the roadmap?